src/co/elastic/NotificationManager.groovy

```diff
@@ -289,7 +289,7 @@ def createGitHubIssue(Map args = [:]) {
     arguments['archiveFile'] = false
     body = generateBuildReport(arguments)
   }
-  def labels = 'automation,ci-reported'
+  def labels = 'automation,ci-reported,ci-build'
```

Comment:
What are the differences between `ci-build` and `ci-reported`? Could they be interchangeable?

Reply:
`ci-reported` is the generic one we have used for the automation regardless of the test-flakiness or build issues. While `ci-build` is the one to filter build related failures (regardless of what the build failure is about)

The idea is to provide some unique github label so people can use queries to filter the kind of the GitHub issues. At the moment is all together

Reply:
I see. What if we went the other direction and specifically made a label for flakiness and then a second label for build failures?

Reply:
```suggestion
  def labels = 'automation,ci-reported,build-failures'
```